src/Record.ts

```diff
@@ -100,16 +128,40 @@ export function collect<A, B>(
 
 /**
  * Get a sorted `Array` of the key/value pairs contained in a `Record`.
+ * Sorted alphabetically by key.
+ *
+ * @example
+ * import { toArray } from 'fp-ts/Record'
+ *
+ * const x = { c: 3, a: "foo", b: false };
+ * assert.deepStrictEqual(toArray(x), [
+ *   ["a", "foo"],
+ *   ["b", false],
+ *   ["c", 3],
+ * ]);
  *
  * @since 2.0.0
  */
 export const toArray: <K extends string, A>(r: Record<K, A>) => Array<[K, A]> =
   /*#__PURE__*/
   collect(S.Ord)((k, a) => [k, a])
 
+// TODO: consider adding an Ord parameter in v3
```

Comment:
This seems to be a buried in the PR, worth converting into a dedicated issue?

Reply:
I guess you're right, I'll remove this from the PR with my next commit.